report-viewer/src/stores/store.ts

```diff
@@ -1,28 +1,33 @@
 import { defineStore } from 'pinia'
-import type { State, SubmissionFile, File, LoadConfiguration } from './state'
+import type { State, SubmissionFile, File, LoadConfiguration, UIState } from './state'
 
 /**
  * The store is a global state management system. It is used to store the state of the application.
  */
 const store = defineStore('store', {
-  state: (): State => ({
-    submissionIdsToComparisonFileName: new Map<string, Map<string, string>>(),
-    anonymous: new Set(),
-    files: {},
-    submissions: {},
-    local: false,
-    zip: false,
-    single: false,
-    fileString: '',
-    fileIdToDisplayName: new Map()
+  state: (): { state: State; uiState: UIState } => ({
```

Comment:
Some of these variable names are really hard to understand as an outsider, e.g., `single` and `fileString`